src/managers/GEventHandling.js

```diff
@@ -197,11 +211,11 @@ class GEventHandling {
                         return msg;
                     },
                     edit: async (options = undefined) => {
-                        if (!botMessage) throw new GError('[NEED RESPOND]',`First you need to send a respond.`);
+                        if (!botMessage) throw new GError('[NEED RESPOND]', `First you need to send a respond.`);
                         let editedMsg = await botMessage.edit(options);
                         return editedMsg;
                     },
-                }, args, objectArgs);
+                }, args, subCommands, objectArgs);
```

Comment:
args, objectArgs, subCommands

Reply:
Maybe this should be like the `commandRunOptions`? So 
```javascript
run({ etc }, { args, subCommands }) {}
```

Reply:
> 
> 
> Maybe this should be like the `commandRunOptions`? It's a little but cluttering if args or objectArgs is not used.
> 
> ```js
> run({ etc }, { args, subCommands }) {}
> ```

So we solve this by making args and objectArgs into the first run object (CommandRunOptions) together with subCommand in 7.0.0 and the second part will be deprecated because it doesn't make sense to make it run({ etc }, args, objec)

If someone doesn't like the fact that they have to have an args variable, they can make `let etc = args;`

Reply:
> 
> 
> Maybe this should be like the `commandRunOptions`? It's a little but cluttering if args or objectArgs is not used.
> 
> ```js
> run({ etc }, { args, subCommands }) {}
> ```

So we solve this by making args and objectArgs into the first run object (CommandRunOptions) together with subCommand in 7.0.0 and the second part will be deprecated because it doesn't make sense to make it run({ etc }, args, objec)

If someone doesn't like the fact that they have to have an args variable, they can make `let etc = args;`